src/main/java/com/easyrest/components/owner/CreateNewEmployee.java

```diff
@@ -0,0 +1,54 @@
+package com.easyrest.components.owner;
+
+import com.easyrest.components.BaseComponent;
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+public class CreateNewEmployee {
+    private WebDriver driver;
+    private final By nameField = By.xpath("//input[@name='name']");
+    private final By emailField = By.xpath("//input[@name='email']");
+    private final By passwordField = By.xpath("//input[@name='password']");
+    private final By phoneField = By.xpath("//input[@name='phone_number']");
+    private final By addButton = By.xpath("//*[text() = 'Add']");
+    private final By cancelButton = By.xpath("//*[text() = 'Cancel']");
+
+    public CreateNewEmployee(WebDriver driver) {
+        this.driver = driver;
+    }
+
+    //We know that we have code duplication
+    public CreateNewEmployee sendToFieldName(String name) {
```

Comment:
maybe named method "inputName"to make it more simple?

Reply:
the same comment for 3 methods below

Reply:
yes, inputName it more simple but sendToFieldName has more description what we do